golf_vehicle_description/urdf/vehicle.xacro

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <!-- load parameter -->
+  <xacro:property
+    name="vehicle_info"
+    value="${load_yaml('$(find golf_vehicle_description)/config/vehicle_info.param.yaml')}"
+  />
+
+  <!-- vehicle body -->
+  <link name="base_link">
+    <visual>
+      <!--<origin xyz="${vehicle_info['/**']['ros__parameters']['wheel_base']/2.0} 0 0" rpy="${pi/2.0} 0 ${pi/2.0}"/>-->
+      <origin xyz="1.3772 0 -0.1" rpy="${pi/2.0} 0 -${pi/2.0}"/>
```

Comment:
There is still golf exceeds ground slightly,I think this will solve our problem:
```suggestion
      <origin xyz="1.3772 0 0" rpy="${pi/2.0} 0 -${pi/2.0}"/>
```

Reply:
I have committed your suggestion. can you check the vehicle again?